tests-integration/build.rs

```diff
@@ -117,6 +117,75 @@ fn main() -> io::Result<()> {
         }
     }
 
+    // Roundtrip Tests
+    let dest_roundtrip = Path::new(&out_dir).join("gen_tests_roundtrip.rs");
+    let mut f = File::create(dest_roundtrip)?;
+    let test_dir = "tests/roundtrip";
+
+    for subdir in WalkDir::new(test_dir) {
+        let subdir = subdir?;
+        // Checks every subdirectory
+        if subdir.file_type().is_dir() {
+            println!("Subdir {}", subdir.file_name().display());
```

Comment:
should this print statement be here? if we want it, perhaps we should hide if behind a verbosity flag (though I suggest we just remove it)